.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,43 @@
+# Modified for MoinMoin, copied from: https://github.com/actions/starter-workflows/blob/main/ci/python-package.yml
+
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  # push:
+  #   branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

Comment:
btw, for borgbackup 1.2 (to be released soon) i decided to drop everything < 3.8.

even the stable dists of misc OSes now have 3.8+ and usually software that is not released yet (== released in the future) does not get into current-stable, but rather into next-stable OS release.

also means:
- less load on / less waiting for CI server.
- ability to use recent python features

Reply:
fixed in try 2